lesson05/assignment/css/style.scss

```diff
@@ -1,6 +1,264 @@
 // Import partials here
-@import 'base.normalize';
-@import 'settings.variables';
-@import 'settings.responsive';
-@import 'components.content';
+@import "base.normalize";
+@import "settings.variables";
+@import "settings.responsive";
+@import "components.content";
 
+
+.wrapper {
+    display: grid;
+    grid-template-columns: 2fr 5fr 2fr;
+    grid-template-rows: 10fr 20fr 5fr;
+    grid-gap: 1vh;
+    height: 150vh;
+}
+
+/*********************
+Header and Hero Image and Placement
+**********************/
+header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column-start: 1;
+    grid-column-end: 4;
+    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("cherry.jpg");
+    background-size: cover;
+    position: relative;
+
+}
+
+.hero-text {
+    text-align: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+  }
+
+
+
+
+/*********************
+Contents and Hero Image and Placement
+**********************/
+.items {
+    background: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 1px;
+    margin-right: 1px;
+    text-align: justify;
+}
+
+.contents {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: justify;
+}
+
+.button {
+    background-color: #4CAF50; /* Green */
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: .5em;
+    cursor: pointer;
+  }
+
+.button-container{
+    display: flex;
+    flex-direction: row;
+    margin: 1em;
+}
+
+img {
+    max-width: 100%;
+}
+/*******************************/
+/* nav bar 
+*******************************/
+.aclass-top {
+    color: white;
+    font-weight: 300;
+    letter-spacing: 2px;
+    text-decoration: none;
+    /* background: rgb(236, 179, 179); */
+    opacity: 1.0;
+    padding: 20px 5px;
```

Comment:
For almost all styling, we should be using responsive units (rem, em, %), not static (px).